Telereso/Classes/Extensions/TButtonExtension.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  TButtonExtension.swift
+//  Telereso
+//
+//  Created by Ganesh TR on 04/05/22.
+//
+
+import SDWebImage
+
+extension UIButton {
+
+    public func setRemoteImageWith(key: String, for state: UIControl.State) {
+        guard let url = Telereso.getRemoteDrawable(key: key) else {
+            self.setImage(UIImage(named:key), for: state)
+            return
+        }
+        sd_setImage(with: url, for: state, placeholderImage: nil, options: [.highPriority, .avoidAutoSetImage, .fromCacheOnly]) { [weak self] cachedImage, error, cacheType, url in
```

Comment:
we should expose optional `placeholderImage` to allow devs to provide there own if needed